CppPluginSupport/src/MfxMesh.cpp

```diff
@@ -78,10 +80,12 @@ MfxAttribute MfxMesh::AddMeshAttribute(const char* name, int componentCount, con
 	return AddAttribute(kOfxMeshAttribMesh, name, componentCount, type);
 }
 
-void MfxMesh::Allocate(int pointCount, int vertCount, int faceCount)
+void MfxMesh::Allocate(int pointCount, int vertCount, int faceCount, int noLooseEdge, int constantFaceCount)
```

Comment:
`noLooseEdge` should be a bool imho

Reply:
Good point, I wrote this before I made it a bool in the struct; I'll change this to bool.